support/openharmony/entry/oh-package.json5

```diff
@@ -0,0 +1,12 @@
+{
+  "license": "",
```

Comment:
We should probably add the license here. Can we have keep the ArkTS app Apache-2 licensed?

Reply:
Done in [e0b51fb](https://github.com/servo/servo/pull/33295/commits/e0b51fb229acbabdb6d2f192cea530fa7d151538)